packages/wallets/src/features/cashier/modules/WithdrawalCrypto/utils/__tests__/withdrawalCryptoValidators.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { THooks } from '../../../../../../types';
+import { validateCryptoAddress, validateCryptoInput, validateFiatInput } from '../withdrawalCryptoValidators';
+
+describe('withdrawalCryptoValidator', () => {
+    let mockValue = '2.5';
+    let mockIsClientVerified = true;
+    let mockCryptoAddress = 'jds93e9f8wefun9w8efrn98wefn09inf0';
+
+    const mockActiveWallet = {
+        balance: 10,
+        currency: 'BTC',
+        currency_config: {
+            minimum_withdrawal: 1,
+        },
+    } as THooks.ActiveWalletAccount;
+
+    const mockFractionalDigits = {
+        crypto: 7,
+        fiat: 2,
+    };
+    const mockRemainder = 9;
+
+    it('should check if no errors are returned when valid inputs are provided for crypto address, crypto amount and fiat amount', () => {
```

Comment:
for future: typically its cleaner to validate only one function / class / compnent in one unit case. Unit tests are cheap so it does not matter if there is more of them. Easier to fix / debug / investigate when they are separate (e.g. if that test fails, I do not know which one out of those 3 functions I broke without prior debugging and investigation).

classic pattern "given - when - then" is a good rule of thumb (unless for given case it make more sense to do something else for whatever reason)

Reply:
Noted, will make this change.